src/components/Summary/AlterationSummary.vue

```diff
@@ -197,6 +219,12 @@ export default class AlterationSummary extends Mixins(CommonMixin, DateMixin, Fi
     return (this.pleaseValidate && !this.getEffectiveDateTime.valid)
   }
 
+  /** Local getter, using a mixin method to detect changes to Share Structure. */
+  private get hasShareStructureChanged (): boolean {
+    return !this.isSame(this.getShareClasses, this.getSnapshotShareStructure?.shareClasses, ['action'])
```

Comment:
I don't think the conditional chaining should be needed here. Isn't `snapshotShareStructure` initialized in the store? So it can never be null/undefined?

Reply:
You are correct!

Reply:
Fixed